### DISCUSSION TITLE
Problem with npm install and wrong python path

### DISCUSSION THREAD

**Discussion by aizure** [community/community#34648] [Packages]
Hi, already sorry if i selected the category wrongfully. I'm having an issue with instalation of ANY module/package using npm install. As during installation npx is trying to verify python3 path, but it doesn't exist as the default path name for python is well.. python.
Let me provide an example:
`npm install ytdl-core
npm ERR! code 1
npm ERR! path C:\Users\ja\Desktop\VarioxaNew3\node_modules\youtube-dl-exec
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c npx bin-version-check-cli python3 "> =3.7"
npm ERR! Error: Couldn't find version of `python3``

I already tried to add python3 in npm config but it didn't do anything:
`python = "C:\\Users\\ja\\AppData\\Local\\Programs\\Python\\Python310\\python.exe"
python3 = "C:\\Users\\ja\\AppData\\Local\\Programs\\Python\\Python310\\python.exe"`

I'm quite unsure what i can do about this..

**Comment by jfullstackdev** [comment]
maybe this will help 

https://stackoverflow.com/questions/67399785/how-to-solve-npm-install-error-npm-err-code-1
